dev-libs/sblim-sfcc/sblim-sfcc-2.2.8.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Small Footprint CIM Client Library"
+HOMEPAGE="https://sourceforge.net/apps/mediawiki/sblim"
+SRC_URI="mirror://sourceforge/project/sblim/${PN}/${P}.tar.bz2"
+
+LICENSE="EPL-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	net-misc/curl[ssl]
+	"
+RDEPEND="${DEPEND}"
+DOCS=()
+
+src_configure() {
+	local myconf=(
+		--enable-http-chunking
+		--disable-static
```

Comment:
this option does not work

Reply:
Hmm, not sure what's wrong with it honestly

Reply:
the package has `--enable-static` and `--enable-shared` options, you must arrange  them properly.

Reply:
See below.